README.md

```diff
@@ -149,12 +149,12 @@ one that uses [Stripe Connect][connect], it's also possible to set a
 per-request key and/or account:
 
 ```php
-$customers = \Stripe\Customer::all([],[
+$customers = $stripe->customers->all([],[
     'api_key' => 'sk_test_...',
     'stripe_account' => 'acct_...'
 ]);
 
-\Stripe\Customer::retrieve("cus_123456789", [
+$stripe->customers->retrieve('cus_123456789', [
```

Comment:
Nit, this should be:
```suggestion
$stripe->customers->retrieve('cus_123456789', [], [
```

The `retrieve()` method on services accepts 3 arguments:
- the ID of the resource to retrieve
- request parameters
- request options